ballista/rust/executor/executor_config_spec.toml

```diff
@@ -78,3 +78,21 @@ name = "task_scheduling_policy"
 type = "ballista_core::config::TaskSchedulingPolicy"
 doc = "The task scheduing policy for the scheduler, see TaskSchedulingPolicy::variants() for options. Default: PullStaged"
 default = "ballista_core::config::TaskSchedulingPolicy::PullStaged"
+
+[[param]]
+name = "executor_cleanup_enable"
+type = "bool"
+doc = "Enable periodic cleanup of work_dir directories."
+default = "true"
```

Comment:
Can we set the default value to false and open it after a long time of testing?
@Ted-Jiang 
If the default is false, this feature will not impact the current status. Users who want to use this feature can open it.
Do you agree? @alamb

Reply:
I think it would be a good idea to put this feature in turned off by default so it can be tested more thoroughly